scipy/stats/_resampling.py

```diff
@@ -2132,9 +2132,21 @@ class ResamplingMethod:
         the statistic. Batch sizes >>1 tend to be faster when the statistic
         is vectorized, but memory usage scales linearly with the batch size.
         Default is ``None``, which processes all resamples in a single batch.
+    random_state : {None, int, `numpy.random.Generator`,
+                    `numpy.random.RandomState`}, optional
```

Comment:
@melissawm I think I saw you fixing one of these at the summit - how do you make a line break not change the formatting?

Or should I just include this all on one line and add the `noqa` at the end of the docstring?

Reply:
Here's how: 

```suggestion
    random_state : {None, int, `numpy.random.Generator`, \
    `numpy.random.RandomState`}, optional
```

This works, but breaks the line for the rendered output as well:
![Captura de imagem_20240619_191040](https://github.com/scipy/scipy/assets/3949932/85950a05-b762-4c78-9e74-8b1f25867744)

Reply:
Hmm. Looks like that just makes it part of the text description. ("optional" is regular font.)

Reply:
I was pretty sure I had this figured out but now can't find the commit. Let me look again

Reply:
x-ref https://github.com/numpy/numpydoc/issues/87#issuecomment-1832867402

Reply:
Just found the same ☝🏻  thanks @lucascolley I think that's exactly what we're hitting 😢

Reply:
I would consider naming this rng as it's going to be the new name for it as per the SPEC7. That would mean that you would directly only support the new way, but IMO a better user story as people will not need to update their code vs now they might use update it once for this and then again to have the new behavior.

https://github.com/scientific-python/specs/issues/180